### PR TITLE
docs: add questions to FAQ

### PR DESCRIPTION
**Pull request by barriebyron**
This PR adds general knowledge questions to the FAQ.

I have a question about the doc title: o1js FAQ 

1. Should we change the name to zkApp and o1js FAQ?
2. Or do we want two FAQ files? 

We'll iterate on adding more useful FAQs as questions are answered

### CODE REVIEW

File: docs/zkapps/faq.mdx [hunk 1]
```diff
@@ -1,15 +1,35 @@
...
-  - proof generation
+  - zkApps
 ---
 
+## What files do I use to write zkApps?
+
+There are many approaches to building a smart contract. For the zkApp tutorials, most examples follow this convention:
+
+- `index.ts`: The entry point of your project that imports all smart contract classes you want access to and exports them to your smart contract. 
+- `main.ts`: How you interact with the smart contract. For example, the `import` statement brings in items from `o1js` that you use to interact with your smart contract.
```
Comment (Trivo25) [inline docs/zkapps/faq.mdx:17]: I would rephrase `items` to maybe methods or objects from o1js

**Review by Trivo25 — APPROVED** [review]
lgtm

File: docs/zkapps/faq.mdx [hunk 2]
```diff
@@ -1,15 +1,35 @@
 ---
 title: o1js FAQ
```
Comment (barriebyron) [inline docs/zkapps/faq.mdx:2]:
```suggestion
title: o1js and zkApps FAQ
```

File: docs/zkapps/faq.mdx (hunk 1 above)
Comment (barriebyron) [inline docs/zkapps/faq.mdx:17]:
```suggestion
- `main.ts`: How you interact with the smart contract. For example, the `import` statement brings in objects and methods from `o1js` that you use to interact with your smart contract. 
```
